tests/unit/LocalDB.js

```diff
@@ -253,9 +253,10 @@ define([
 					]);
 				});
 			},
-			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))":
-				testQuery(new Filter().in('mappedTo', letterStore.filter({vowel: true}).select('id')),
-					{sort:[{property: 'name'}], range: {start: 1, end: 3}}, [5, 1]),
+			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))": (function() {
```

Comment:
No it doesn't. That was changed for debugging purposes because it was failing. I've changed it back.